.gitignore

```diff
@@ -110,4 +110,7 @@ templates/layout
 tags
 
 # Ignore wdio/chromedriver logs
-tests/functional/logs/
\ No newline at end of file
+tests/functional/logs/
+
+# `typing` folder
```

Comment:
Not sure this comment adds much, I still don't really know what would be in typing to ignore